docs/source/api/v4/cdns_id_notification.rst

```diff
@@ -0,0 +1,123 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-cdns-id-notification:
+
+****************************
+``cdns/{{ID}}/notification``
+****************************
+
+``POST``
+========
+Create a notification for a specific CDN.
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object
+
+Request Structure
+-----------------
+.. table:: Request Path Parameters
+
+	+------+-------------------------------------------------------------------------------+
+	| Name | Description                                                                   |
+	+======+===============================================================================+
+	| ID   | The integral, unique identifier for the CDN on which to create a notification.|
+	+------+-------------------------------------------------------------------------------+
+
+:notification: The content of the CDN notification
+
+.. code-block:: http
+	:caption: Request Example
+
+	POST /api/4.0/cdns/2/notification HTTP/1.1
+	Host: trafficops.infra.ciab.test
+	User-Agent: curl/7.47.0
+	Accept: */*
+	Cookie: mojolicious=...
+	Content-Length: 19
+	Content-Type: application/json
+
+	{"notification": "No changes to Traffic Ops over the weekend."}
+
+Response Structure
+------------------
+:cdnId:			The integral, unique identifier for the CDN on which :term:`Queue Updates` was performed or cleared
+:notification:	The notification added to the CDN
+:username:		The username of the user responsible for creating the CDN notification
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Access-Control-Allow-Credentials: true
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Type: application/json
+	Set-Cookie: mojolicious=...; Path=/; Expires=Mon, 18 Nov 2019 17:40:54 GMT; Max-Age=3600; HttpOnly
+	Whole-Content-Sha512: rBpFfrrP+9IFkwsRloEM+v+I8MuBZDXqFu+WUTGtRGypnAn2gHooPoNQRyVvJGjyIQrLXLvqjEtve+lH2Tj4uw==
+	X-Server-Name: traffic_ops_golang/
+	Date: Wed, 14 Nov 2018 21:02:07 GMT
+	Content-Length: 41
+
+	{ "response": {
+		"cdnId": 2,
```

Comment:
I have to know the CDN ID to call this endpoint, I'm not sure I need this in the response

Reply:
gone